sdcm/tester.py

```diff
@@ -2584,49 +2587,82 @@ def copy_data_between_tables(self, node, src_keyspace, src_table, dest_keyspace,
         self.log.debug('All rows have been copied from %s to %s', src_table, dest_table)
         return True
 
-    def collect_partitions_info(self, table_name, primary_key_column, save_into_file_name) -> dict | None:  # pylint: disable=too-many-locals
+    def validate_partitions(self, limit_total_rows_number: bool = True):
```

Comment:
why is `limit_total_rows_number` needed? please add few words about it in the function's docstring

Reply:
fixed

Reply:
renamed to `skip_on_rows_number_limit`.
@soyacz , as for the alternative approach, it's possible but i think it might be confusing. because it won't really validate all data. It will only check an arbitrary small portion of the data.
And secondly, even a single partition could have a size of 10M rows, and will cause and additional high read stress.
And in any case, all data will be validation at the end of test.